src/book/02-system/01-foundations/images/maths-spline.svg

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0"?>
```

Comment:
Could pass this through SVGOMG to reduce the size: https://jakearchibald.github.io/svgomg/